creator-node/src/routes/files.js

```diff
@@ -12,6 +12,188 @@ const { authMiddleware, syncLockMiddleware, triggerSecondarySyncs } = require('.
 const { getIPFSPeerId, rehydrateIpfsFromFsIfNecessary, ipfsSingleByteCat } = require('../utils')
 const ImageProcessingQueue = require('../ImageProcessingQueue')
 
+// Helper method to stream file from file system on creator node
+const streamFromFileSystem = async (req, res, path) => {
+  try {
+    // If file cannot be found on disk, throw error
+    if (!fs.existsSync(path)) {
+      return sendResponse(req, res, errorResponseServerError('File could not be found on disk.'))
+    }
+
+    // Stream file from file system
+    let fileStream
+
+    // If a range header is present, use that to create the readstream
+    // otherwise, stream the whole file.
+    const range = req.range()
+    if (range && range[0]) {
+      const { start, end } = range[0]
+      fileStream = fs.createReadStream(path, { start, end })
+    } else {
+      fileStream = fs.createReadStream(path)
+    }
+
+    await new Promise((resolve, reject) => {
+      fileStream
+        .on('open', () => fileStream.pipe(res))
+        .on('end', () => { res.end(); resolve() })
+        .on('error', e => { reject(e) })
+    })
+  } catch (e) {
+    // Unable to stream from file system. Throw a server error message
+    return sendResponse(req, res, errorResponseServerError(e.message))
```

Comment:
fyi we just changed this to not send response but throw error instead, per the latest on master

Reply:
thanks dawg

Reply:
fixed!